tools/stress-cluster/chaos/examples/network-stress-example/Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM alpine:3.14
-RUN apk add --no-cache wget
-RUN apk add --no-cache bash
+FROM mcr.microsoft.com/cbl-mariner/base/core:1.0
```

Comment:
Let's not bother pinning a version and just consume latest, that way we'll have fewer security notifications going forward.

Reply:
![image](https://user-images.githubusercontent.com/38804567/165636928-064ea094-5fe1-4481-979e-974a1b25fa18.png)
I don't think they allow us to use latest?

Reply:
There's a trailing colon in that? You could also omit any tag which should default to latest.

Reply:
There is also no `latest` tag: https://mcr.microsoft.com/v2/cbl-mariner/base/core/tags/list

Reply:
It is better to use base image on 2.0 instead of 1.0. Mariner 1.0 is about to stop maintenance. 
https://eng.ms/docs/products/mariner-linux/gettingstarted/containers/marinercontainerimage#nodejs-image